Tracing: Add ability to write trace to metrics query (#48824)
grafana/grafana: docs/sources/datasources/zipkin.md

![Trace to logs settings](/static/img/docs/explore/trace-to-logs-settings-8-2.png 'Screenshot of the trace to logs settings')

### Trace to metrics

> **Note:** This feature is behind the `traceToMetrics` feature toggle.

> This is a configuration for the experimental trace to metrics feature. Select the target Prometheus data source and enter the desired query.

Reviewer: ```suggestion
To configure trace to metrics, select the target Prometheus data source and enter the desired query.
```

-- **Data source -** Target data source.

> -- **Query -** The query which will run when navigating from a trace to the metrics data source.

Reviewer: ```suggestion
-- **Query -** Query that runs when navigating from a trace to the metrics data source.
```
